public/index.css

```
@@ -0,0 +1,272 @@
+html {
```

нужно вернуть то что было раньше
```suggestion
* {
```
```suggestion
html {
```